dali/python/backend_impl.cc

```diff
@@ -705,9 +762,46 @@ py::tuple TensorListGetItemSliceImpl(TensorList<Backend> &t, py::slice slice) {
 }
 #endif
 
+template <typename Backend>
+using backend_to_py_class_t = py::class_<TensorList<Backend>, std::shared_ptr<TensorList<Backend>>>;
```

Comment:
nitpick: For me the name is confusing: it does not mention tensorlist at all suggesting that it is some generic utility that transforms backend into py_class (how about just tensor_list_py_class_t or sth along those lines?

Reply:
done